roles/static_inventory/tasks/sshtun.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Create ssh tunnel systemd service
+  template:
+    src: ssh-tunnel.service.j2
+    dest: /etc/systemd/system/ssh-tunnel.service
+    mode: 0755
```

Comment:
Huh, strange. This seems to suggest that they should be 644 instead:

https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/sect-Managing_Services_with_systemd-Unit_Files.html#proc-Managing_Services_with_systemd-Unit_File_Create

Would you mind trying to see if that works? Could be something weird on your system. But we can also do that as a separate patch if you prefer.